### PR TITLE
[FIX] mail: functions for FieldCommand should return boolean value

### PR DESCRIPTION
**Pull request by qho-odoo**
The executing function for FieldCommand should return boolean value 
to indicate whether the value changed for triggering the re-computation.

### CODE REVIEW

File: addons/mail/static/src/model/model_field_command.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ odoo.define('mail/static/src/model/model_field_command.js', function (require) {
 class FieldCommand {
     /**
      * @constructor
-     * @param {function} func function to call when executing this command
+     * @param {function} func function to call when executing this command. The function should ALWAYS return a boolean value to indicate if the field value is changed or not
```
Comment (seb-odoo) [inline addons/mail/static/src/model/model_field_command.js:12]: Just a remark: we follow max 80 char per line when it's possible.
We are less strict with actual code, but it's always possible and easy to keep it under 80 when writing comment.

Same in the commit message.

Comment (seb-odoo) [inline addons/mail/static/src/model/model_field_command.js:12]: Also don't forget to add a dot at the end of the phrase.
And I would just reformulate "if the field value is changed or not" -> "whether the value changed."

Comment (qho-odoo) [reply addons/mail/static/src/model/model_field_command.js]: fixed

Comment (qho-odoo) [reply addons/mail/static/src/model/model_field_command.js]: fixed

**Review by seb-odoo — APPROVED** [review]
Perfect, thank you.

@robodoo r+

**Comment by seb-odoo** [comment]
Runbot all broken? Maybe need to rebase?

**Comment by seb-odoo** [comment]
I think your enterprise branch is out of sync with community.

Also for next time, by convention (don't need to make a new one now but) we usually use - and not _ to add trigram at the end of branch name

**Comment by qho-odoo** [comment]
Branch name should start with [version_name]-[the_rest]
check the new PR #66648

**Pull request closed without merging**